lib/config/resource-type-by-ext.js

```diff
@@ -4,5 +4,7 @@ var types = require('./resource-types');
 module.exports = {
 	'.html': types.html,
 	'.htm': types.html,
-	'.css': types.css
+	'.css': types.css,
+	'.js': types.js,
+	'.min.js': types.js
```

Comment:
I believe `.min.js` is not needed here, such files will have extension`.js`

Reply:
Makes sense..will get rid of this one. I was not sure, I added it to be double sure and cover all bases.

Reply:
Also, I will add the test for .js files and push the code up. You should have a new commit/PR to approve soon!

Reply:
@s0ph1e : The updated code has been pushed and the new commit is in and ready for approval.